components/infobox/wikis/brawlstars/infobox_league_custom.lua

```diff
@@ -0,0 +1,170 @@
+---
+-- @Liquipedia
+-- wiki=brawlstars
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local League = require('Module:Infobox/League')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+local Tier = require('Module:Tier')
+local Class = require('Module:Class')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local PrizePoolCurrency = require('Module:Prize pool currency')
+
+local _TODAY = os.date('%Y-%m-%d', os.time())
+
+local _args
+local _league
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+function CustomLeague.run(frame)
+	local league = League(frame)
+	_league = league
+	_args = _league.args
+
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+	league.addToLpdb = CustomLeague.addToLpdb
+	league.getWikiCategories = CustomLeague.getWikiCategories
+
+	return league:createInfobox(frame)
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	local args = _args
+	table.insert(widgets, Cell{
+		name = 'Teams',
+		content = {(args.team_number or '') .. (args.team_slots and ('/' .. args.team_slots) or '')}
+	})
+
+	return widgets
+end
+
+function CustomInjector:parse(id, widgets)
+	local args = _args
+	if id == 'prizepool' then
+		return {
+			Cell{
+				name = 'Prize pool',
+				content = {CustomLeague:_createPrizepool()}
+			},
+		}
+	elseif id == 'liquipediatier' then
+		if args['supercell-sponsored'] == 'true' then
+			return {
+				Cell{
+					name = 'Liquipedia Tier',
+					content = {(CustomLeague:_createLiquipediaTierDisplay() or '') .. '&nbsp;' ..
+								'[[File:Supercell icon.png|x18px|link=Supercell|Tournament sponsored by Supercell.]]'},
+					classes = {'valvepremier-highlighted'}
```

Comment:
Is this right?

Reply:
css class is called that, yup
https://liquipedia.net/brawlstars/index.php?title=Template:Infobox_league&action=edit#mw-ce-l204